### ISSUE TITLE
Error in case-sensitive system

### ISSUE THREAD

**Issue by Phyks** [David-Desmaisons/Vue.ImagesLoaded#1]
Hi,

```
require("imagesLoaded")
```
in this module seems to be failing on case-sensitive filesystems (and should be `require(imagesloaded)` instead I guess).

Webpack build is working on a Mac, but not working on Linux, which seems to be the same as [this SO post](https://stackoverflow.com/questions/13617863/can-i-force-node-js-require-to-be-case-sensitive).

A workaround for users at the moment (if anyone has the same error and comes across this issue) is to use:
```
module.exports = {
    resolve: {
            'imagesLoaded': 'imagesloaded'
    }
}
```
in the webpack config.

Thanks!

**Comment by David-Desmaisons** [comment]
Thanks for pointing out that.
This is corrected in version 1.1.2.

**Comment by Phyks** [comment]
Awesome! Thanks!
